packages/common/src/events/_internal/EventSchemaEdit.ts

```diff
@@ -67,7 +68,17 @@ export const buildSchema = (): IConfigurationSchema => {
         default: HubEventCapacityType.Unlimited,
       },
       location: {
-        type: "object",
+        ...ENTITY_LOCATION_SCHEMA,
+        allOf: [
```

Comment:
for events, we add this condition to the schema so that the overall form is registered as "invalid" if custom/org is selected and a location name is not provided